api/controllers/UserController.js

```diff
@@ -254,7 +254,7 @@ module.exports = {
         return res.status(400).json({error: "Invalid friendcode list"});
       }
     }
-
+    console.log("/u/" + req.user.name + ": Started process to ban /u/" + req.params.username);
```

Comment:
We should really get a proper logger.

Then we can do different logs for console and events, would be good.